src/frontend/web_application/src/scenes/MessageList/components/ReplyForm/presenter.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { Component, PropTypes } from 'react';
+import ReplyFormBase from '../../../../components/ReplyForm';
+
+class ReplyForm extends Component {
+  static propTypes = {
+    discussionId: PropTypes.string.isRequired,
+    message: PropTypes.shape({ }),
+    draft: PropTypes.shape({ }),
+    requestDraft: PropTypes.func.isRequired,
+    editDraft: PropTypes.func.isRequired,
+    saveDraft: PropTypes.func.isRequired,
+    sendDraft: PropTypes.func.isRequired,
+  };
+
+  static defaultProps = {
+    message: {},
+    draft: null,
+  };
+
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+    this.handleSend = this.handleSend.bind(this);
+  }
+
+  componentWillMount() {
+    const { discussionId, draft } = this.props;
+    if (!draft && discussionId) {
+      this.props.requestDraft({ discussionId });
+    }
+  }
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
Actually `requestDraft` doesn't fetch drafts since there's no API for drafts: draft are common message with `is_draft: true`; 
the `requestDraft` will filter the `message` state and initialize a draft in the state (a new one if there is no existing draft message).